packages/hoppscotch-js-sandbox/src/apis/env/utils.ts

```diff
@@ -0,0 +1,70 @@
+import * as O from "fp-ts/Option"
+import cloneDeep from "lodash/cloneDeep"
+import { TestScriptReport } from "../../test-runner"
+
+export function deleteEnv(envName: string, envs: TestScriptReport["envs"]) {
+  const updatedEnvs = cloneDeep(envs)
+  const indexInGlobal = envs.global.findIndex((x) => x.key === envName)
+  const indexInSelected = envs.selected.findIndex((x) => x.key === envName)
+
+  updatedEnvs.global.splice(indexInGlobal, 1)
+  updatedEnvs.selected.splice(indexInSelected, 1)
+
+  return updatedEnvs
+}
+
+export function getActiveEnv(envName: string, envs: TestScriptReport["envs"]) {
```

Comment:
A better name will be `getFromActiveEnv` and instead of `envName` a better name would be `varName` as we are giving it the name of the variable to access. Not the environment

Reply:
How about `getActiveFromEnv` ????